### PR TITLE
External link V2

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes https://github.com/sparkletown/internal-sparkle-issues/issues/1227

### CODE REVIEW

File: src/components/templates/ExternalRoom/ExternalRoom.scss [hunk 1]
```diff
@@ -1,43 +1,110 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: $spacing;
+  }
+
+  &__venue-logo {
+    flex-shrink: 0;
+    height: 180px;
+    width: 180px;
```
Comment (yurii-lubynets) [inline src/components/templates/ExternalRoom/ExternalRoom.scss:78]: Let's extract these sizes as vars

File: src/pages/RegistrationFlow/LogoRF/LogoRF.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import "./LogoRF.scss";
+
+export const LogoRF = () => {
```
Comment (sunny-viktoryia) [inline src/pages/RegistrationFlow/LogoRF/LogoRF.tsx:3]: can you export a single image instead of placing 4 of them?

File: src/components/templates/ExternalRoom/ExternalRoom.tsx [hunk 3]
```diff
@@ -17,46 +27,88 @@ export interface ExternalRoomProps {
...
+                <div className="ExternalRoom__venue-details">
+                  <div className="ExternalRoom__venue-title">{venue.name}</div>
+
+                  <div className="ExternalRoom__venue-subtitle">
+                    {venue.config?.landingPageConfig.subtitle}
+                  </div>
+
+                  <ButtonNG
+                    variant="primary"
+                    onClick={() => openUrl(redirectUrl)}
```
Comment (sunny-viktoryia) [inline src/components/templates/ExternalRoom/ExternalRoom.tsx:88]: useCallback ?

**Review by sunny-viktoryia — COMMENTED** [review]
looks good. A couple of nit comments and then should be good to get merged. 🙃

File: src/components/templates/ExternalRoom/ExternalRoom.scss (hunk 1 above)
Comment (alexey-petrov) [reply src/components/templates/ExternalRoom/ExternalRoom.scss]: fixed 848c7e3

File: src/pages/RegistrationFlow/LogoRF/LogoRF.tsx (hunk 2 above)
Comment (alexey-petrov) [reply src/pages/RegistrationFlow/LogoRF/LogoRF.tsx]: fixed 3b34439

File: src/components/templates/ExternalRoom/ExternalRoom.tsx (hunk 3 above)
Comment (alexey-petrov) [reply src/components/templates/ExternalRoom/ExternalRoom.tsx]: done 3b34439

File: src/pages/RegistrationFlow/LogoRF/LogoRF.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import "./LogoRF.scss";
+
+export const LogoRF = () => (
```
Comment (sunny-viktoryia) [inline src/pages/RegistrationFlow/LogoRF/LogoRF.tsx:3]: and why is this logo is in the registration flow? what way is that related to the external link?

Comment (alexey-petrov) [reply src/pages/RegistrationFlow/LogoRF/LogoRF.tsx]: wrong location, moved it to organisms fc594aa

File: src/components/atoms/SparkleLogoIcon/SparkleLogoIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import "./SparkleLogoIcon.scss";
+
+export const SparkleLogoIcon = () => (
+  <div className="SparkleLogoIcon">
+    <img
+      className="SparkleLogoIcon__logo"
+      src="/sparkle-logo-rf.svg"
```
Comment (sunny-viktoryia) [inline src/components/atoms/SparkleLogoIcon/SparkleLogoIcon.tsx:7]: can you move the image to src/assets so that we could identify if we're missing it at compile time? 

and, please, rename the "-rf" suffix from the name.

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me! thank you! ❤️ 
Left a nit comment. Would be nice if you could change that. 🙃

File: src/components/atoms/SparkleLogoIcon/SparkleLogoIcon.tsx (hunk 5 above)
Comment (alexey-petrov) [reply src/components/atoms/SparkleLogoIcon/SparkleLogoIcon.tsx]: yep, done 5185f9a
